selfdrive/athena/athenad.py

```diff
@@ -790,8 +790,6 @@ def main():
     except (ConnectionError, TimeoutError, WebSocketException):
       conn_retries += 1
       params.remove("LastAthenaPingTime")
-    except socket.timeout:
-      params.remove("LastAthenaPingTime")
```

Comment:
why was this removed?

Reply:
https://docs.python.org/3/whatsnew/3.10.html#socket

socket.timeout and TimeoutError are now the same exception and pylint throws https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/duplicate-except.html

Reply:
we'll have to silent it for now, since C3's will still be running 3.8 for a while after this is merged. can you revert this, silence the pylint warning, and add a good comment explaining this?